README.md

```diff
@@ -1,2 +1,48 @@
-# text-editor
-Development Text editor. No guarantees are given that this works or keeps working.
+# Text Editor
+Rich inline text editing
+
+## Features
+* Edit selected text
+
```

Comment:
* Edit selected text
* HTML output of formatted text
* Show editor options either on a toolbar or as a bubble